### PR TITLE
FNC_Cortex v1.0.1 integration publishing

### PR DESCRIPTION
**Pull request by FortiNDR-Integration**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This Fortinet's integration has being used as a custom integration. As per discussion with PAN, we are submitting it to be published in the Marketplace as a community contribution.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
Hi @FortiNDR-Integration, 
You need to sign CLA [here](https://cla-assistant.io/demisto/content?pullRequest=35829) and fill in the contribution registration form [here](https://docs.google.com/forms/d/e/1FAIpQLSd0V270r75tTy9YbT50irQBUpKK8Wfn1aCUihYNn2rc_LkPpA/viewform).

In addition, since this is a duplicate of gigamon, you'll need to deprecate the gigamon pack, you can add the deprecation to this PR.
See [here](https://xsoar.pan.dev/docs/reference/articles/deprecation-process-and-hidden-packs) about how to deprecate a pack.

One last thing, you'll need to change the support type from partner to community, since Fortinet are not Palo Alto partners.

Let me know if you have any questions.

File: Packs/FortiNDRCloud/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Fortinet FortiNDR Cloud",
+    "description": "Fortinet FortiNDR Cloud allows a fast detection and effective response to active threats.",
+    "support": "partner",
```
Comment (edik24) [inline Packs/FortiNDRCloud/pack_metadata.json:4]: Please change to "community"

**Comment by edik24** [comment]
Hi @FortiNDR-Integration ,
you did commits with users @FortiNDR-Integration and @gti-integration.
So both users need to sign the CLA.
Currently as you can see [here](https://github.com/demisto/content/pull/35829#issuecomment-2282812546), only one of them signed.
Can you please sign with both?

File: Packs/FortiNDRCloud/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (thefrieddan1) [inline Packs/FortiNDRCloud/ReleaseNotes/1_0_1.md:1]: Why is this file empty?

**Comment by edik24** [comment]
@FortiNDR-Integration can you sign the CLA?

**Comment by edik24** [comment]
They won't sign the CLA.
Closing.

**Pull request closed without merging**